specs/core/0_beacon-chain.md

```diff
@@ -166,6 +166,7 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 | `BLS_WITHDRAWAL_PREFIX_BYTE` | `0x00` | - |
 | `MAX_CASPER_VOTES` | `2**10` (= 1,024) | votes |
 | `LATEST_BLOCK_ROOTS_LENGTH` | `2**13` (= 8,192) | block roots |
+| `LATEST_RANDAO_MIXES_LENGTH` | `2**13` (= 8,192) | block roots |
```

Comment:
Unit is not "block roots". 
Use "randao mixes" or "seeds" if you keep a unit here.

Reply:
Done! (Also done the other changes below).